pkgs/applications/misc/cataract/build.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, fetchgit
+, autoconf
+, automake
+, glib
+, pkgconfig
+, libxml2
+, exiv2
+, imagemagick
+, version
+, sha256
+, rev }:
+
+stdenv.mkDerivation rec {
+  inherit version;
+  name = "cataract-${version}";
+
+  src = fetchgit {
+    url = "git://git.bzatek.net/cataract";
+    inherit sha256;
+    inherit rev;
+  };
+
+  buildInputs = [ autoconf automake glib pkgconfig libxml2 exiv2 imagemagick ];
```

Comment:
`autoreconfHook` can probably replace `autoconf` and `automake` as well as `./autogen.sh` in buildPhase. This would make `buildPhase` obsolete as well.